virtualmachine/gcp/vm.go

```diff
@@ -36,6 +44,7 @@ var (
 type VM struct {
 	Name        string
 	Description string
+	Region      string
```

Comment:
Isn't zone sufficient?
We get anytime get zone-region mapping for specific cases, if any.

Reply:
Region is required only for GetSubnetworkList() call. Yes, we can get region if the zone is known. I will remove from this struct and use zone, to get region.